commons/src/main/java/org/frankframework/doc/Forward.java

```diff
@@ -32,4 +32,5 @@
 public @interface Forward {
 	String name();
 	String description() default "";
+	boolean wildcard() default false;
```

Comment:
Misschien dat name == '*' ook gebruikt kan worden?

Reply:
Eens. De informatie staat er nu dubbel in.